recipe/install_ucx.sh

```diff
@@ -6,9 +6,9 @@ EXTRA_ARGS=""
 if [ "${cuda_compiler_version}" != "None" ]; then
     EXTRA_ARGS="${EXTRA_ARGS} --with-cuda=${CUDA_HOME}"
 fi
-if [ "${cdt_name}" == "cos6" ]; then
-    EXTRA_ARGS="${EXTRA_ARGS} --with-cm"
-fi
+
+
+# --with-rdmacm requires rdma-core v23+, while CentOS 7 only offers v22.
```

Comment:
I think this is ok for UCX 1.12 and MLNX5.X -- @pentschev can you confirm we can support IB moving forward ?

Reply:
Maybe for MLNX_OFED 4.x, which we (RAPIDS) don't support anymore. MLNX_OFED 5.0 is rdma-core v28. This is the same reason why we still don't support verbs/rdmacm in https://github.com/rapidsai/ucx-split-feedstock .

Reply:
As Matthias has removed the flag and added this comment, I think we are agreeing. That said, is there anything else we should be including in this comment?

Reply:
No, this looks fine to me. If we want to be verbose, we can mention that for MLNX_OFED 5.0 rdma-core v28 is needed, but otherwise this is enough.

Reply:
Thanks for confirming, Peter!